Cartfile

```diff
@@ -1 +1 @@
-github "antitypical/Result" ~> 0.5
+github "antitypical/Result" "9d5cc8edab9af8a25fcd9a2a106c5183f1289898"
```

Comment:
Some further changes are required because of https://github.com/antitypical/Result/pull/69
Would you like to make them here or should I open a separate PR?